packages/localize/schematics/ng-add/index.ts

```diff
@@ -23,14 +23,49 @@ import {
   removePackageJsonDependency,
 } from '@schematics/angular/utility/dependencies';
 import {JSONFile, JSONPath} from '@schematics/angular/utility/json-file';
-import {getWorkspace} from '@schematics/angular/utility/workspace';
+import {getWorkspace, updateWorkspace} from '@schematics/angular/utility/workspace';
 import {Builders} from '@schematics/angular/utility/workspace-models';
 
 import {Schema} from './schema';
 
 const localizeType = `@angular/localize`;
+const localizePolyfill = '@angular/localize/init';
 const localizeTripleSlashType = `/// <reference types="@angular/localize" />`;
 
+function addPolyfillToConfig(projectName: string): Rule {
+  return updateWorkspace((workspace) => {
+    const project = workspace.projects.get(projectName);
+    if (!project) {
+      throw new SchematicsException(`Invalid project name '${projectName}'.`);
+    }
+
+    const isLocalizePolyfill = (path: string) => path.startsWith('@angular/localize');
+
+    for (const target of project.targets.values()) {
+      switch (target.builder) {
+        case Builders.Karma:
+        case Builders.Server:
+        case Builders.Browser:
+        case Builders.BrowserEsbuild:
+        case Builders.Application:
+          target.options ??= {};
+          const value = target.options['polyfills'];
+          if (typeof value === 'string' && !isLocalizePolyfill(value)) {
+            target.options['polyfills'] = [value, localizePolyfill];
+          } else if (Array.isArray(value)) {
```

Comment:
Should we move the localize check inside to avoid overriding the value (string -> array) in case the localize polyfill is already there?
```suggestion
          if (typeof value === 'string') {
            if (!isLocalizePolyfill(value)) {
              target.options['polyfills'] = [value, localizePolyfill];
            }
          } else if (Array.isArray(value)) {
```